livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/realtime/realtime_model.py

```diff
@@ -125,9 +162,13 @@ def __init__(
         temperature: NotGivenOr[float] = NOT_GIVEN,
         base_url: NotGivenOr[str] = NOT_GIVEN,
         input_audio_transcription: NotGivenOr[InputAudioTranscription | None] = NOT_GIVEN,
+        tool_choice: NotGivenOr[llm.ToolChoice | None] = NOT_GIVEN,
```

Comment:
I don't think we need tool_choice in the constructor, I would push against because the Agent isn't going to be aware of it